### PR TITLE
Add command for manupirate configuration file

### PR DESCRIPTION
**Pull request by kuwa72**
Add config command for beginners.
Add descriptions of GODMINE_ENV.

### CODE REVIEW

File: godmine/main.go [hunk 1]
```diff
@@ -695,6 +685,81 @@ func editWikiPage(title string) error {
...
+	"apikey": "%s",
+	"project": %s
+}`
+	content := fmt.Sprintf(template, endpoint, apikey, project)
+
+	file := createConfigFileName()
+	dir := filepath.Dir(file)
+
+	if _, err := os.Stat(dir); err != nil {
+		os.MkdirAll(dir, 0755)
```
Comment (mattn) [inline godmine/main.go]: Sorry this is my bad. Please fix 0700

Comment (kuwa72) [reply godmine/main.go]: Fixed. 
By the way, MkdirAll/WriteFile with permission what about in Windows os? 
```
$ ls -l ~/.config
....
drwx------   3 user1  staff  102 10 27 20:04 godmine
....
```

File: godmine/main.go [hunk 2]
```diff
@@ -695,6 +685,81 @@ func editWikiPage(title string) error {
 	return nil
 }
 
+func initConfigFile(endpoint string, apikey string, project string) error {
+	template := `{
+	"endpoint": "%s",
+	"apikey": "%s",
+	"project": %s
+}`
+	content := fmt.Sprintf(template, endpoint, apikey, project)
```
Comment (mattn) [inline godmine/main.go]: Please do this strictly using encoding/json.

File: godmine/main.go [hunk 3]
```diff
@@ -695,6 +685,81 @@ func editWikiPage(title string) error {
...
+	}
+
+	return ioutil.WriteFile(file, []byte(content), 0600)
+}
+
+func listConfigFile() error {
+	file := createConfigFileName()
+	dir := filepath.Dir(file)
+
+	files, err := ioutil.ReadDir(dir)
```
Comment (mattn) [inline godmine/main.go]: please use os.Readdirnames(0) instead.

**Comment by mattn** [comment]
Thanks your contribution. Mostly LGTM.

File: godmine/main.go [hunk 4]
```diff
@@ -685,55 +687,74 @@ func editWikiPage(title string) error {
...
-	"apikey": "%s",
-	"project": %s
-}`
-	content := fmt.Sprintf(template, endpoint, apikey, project)
+func initConfigFile(endpoint string, apikey string, project string) {
+	u, err := url.Parse(endpoint)
+	if err != nil {
+		fatal("endpoint must be a URL: %s\n", err)
+	} else if (u.Scheme != "http" && u.Scheme != "https") || len(u.Host) == 0 {
+		fatal("%s\n", errors.New("endpoint must be a URL"))
```
Comment (kuwa72) [inline godmine/main.go]: It is hack for error handling adjust to fatal() style. I think its not better.

**Comment by mattn** [comment]
Thank you!

**Pull request merged**